src/Components/Lifecycle/Lifecycle.tsx

```diff
@@ -93,21 +83,22 @@ const LifecycleTab: React.FC<React.PropsWithChildren> = () => {
   const fetchData = async () => {
     setIsLoading(true);
     try {
-      const data = await getLifecycleSystems();
+      const systemData = await getLifecycleSystems();
       const appData = await getLifecycleAppstreams();
-      const upcomingChangesParagraphs = data || [];
+      const upcomingChangesParagraphs = systemData || [];
       const appStreams = appData.data || [];
-      setLifecycleChanges(upcomingChangesParagraphs);
+      setSystemLifecycleChanges(upcomingChangesParagraphs);
       setAppLifecycleChanges(appStreams);
-      const tableData = updateLifecycleData(upcomingChangesParagraphs);
-      setLifecycleChanges(tableData);
+      const updatedSystems = updateLifecycleData(upcomingChangesParagraphs);
+      setSystemLifecycleChanges(updatedSystems);
       if (dropdownValue === DEFAULT_DROPDOWN_VALUE) {
         setFilteredTableData(appStreams);
+        setFilteredChartData(appStreams)
       } else {
-        setFilteredTableData(tableData);
+        setFilteredTableData(updatedSystems);
+        setFilteredChartData(updatedSystems)
       }
     } catch (error) {
-      console.error('Error fetching lifecycle changes:', error);
```

Comment:
We do want to keep this - we're going to want to use it to trigger an error state later. This is a stub.

Reply:
We'll want to log errors to the logging system in any case like this.

Reply:
Noted sorry about that